sdk/src/main/java/com/akkaserverless/javasdk/AkkaServerless.java

```diff
@@ -454,6 +451,16 @@ public AkkaServerless registerValueEntity(
         entityClass, descriptor, ValueEntityOptions.defaults(), additionalDescriptors);
   }
 
+  public <T> AkkaServerless registerValueEntity(ValueEntityFactory<T> factory) {
+    return lowLevel()
```

Comment:
I think we should step out of the idea of having a low-level API. 

Users should not be exposed to such a method and ultimately the default API to use will be the one that does not depend on reflection.

Reply:
agreed, what is the 'lowlevel' API right now would become the 'codegen-based' API and replace the annotation-based API.

Reply:
> I think we should step out of the idea of having a low-level API.

I introduced it, but the intention was never to keep it. It was just too early to remove it and I moved it to the side.